client/src/components/icons/Cross.tsx

```diff
@@ -0,0 +1,23 @@
+function Cross({ color }: { color: string }) {
+  return (
+    <div className="m-1 pt-0.5">
```

Comment:
@schultztimothy I would maybe remove the div and leave only the `svg` element like we have in `Download`, so that it can be used everywhere even where someone doesn't need  a div, wdyt?

Reply:
Good call! I removed these from all of the svg components